raiden/ui/app.py

```diff
@@ -221,7 +188,7 @@ def run_app(
     )
 
     if sync_check:
-        check_synced(blockchain_service, known_node_network_id)
+        check_synced(blockchain_service, network_id)
```

Comment:
As commented before, the second argument here should be a boolean, not the `network_id` itself. This will lead to the check being run when it shouldn't be, in case raiden is run on an unknown private network.